src/webview/markdownEngine.ts

```diff
@@ -107,4 +138,29 @@ class MarkdownEngine {
     }
 }
 
+// tslint:disable-next-line: max-classes-per-file
+class MarkdownConfiguration {
+
+    public readonly extRoot: string; // root path of vscode built-in markdown extension
+    public readonly lineHeight: number;
+    public readonly fontSize: number;
+    public readonly fontFamily: string | undefined;
+
+    public constructor() {
+        const markdownConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("markdown");
+        this.extRoot = path.join(vscode.env.appRoot, "extensions", "markdown-language-features");
+        this.lineHeight = Math.max(0.6, +markdownConfig.get<number>("preview.lineHeight", NaN));
```

Comment:
Why we set the default value to `NaN` here?

Reply:
This is the strategy adopted by [built-in markdown extension](https://github.com/Microsoft/vscode/blob/master/extensions/markdown-language-features/src/features/previewConfig.ts#L45)

I think the logic is as following:
1. if no config for lineHeight is specified, then the result is `Math.max(0.6, NaN) === NaN`, so [line-height](https://github.com/jdneo/vscode-leetcode/pull/260/files#diff-f7ac20b7043db95594bdb6d86f3ef02bR68) property in css won't be set.
2. if the config is set, we should preserve a minimum line height to 0.6.

Reply:
I see...